modules/arch-test/src/test/java/org/apache/ignite/internal/ConfigurationModuleArchTest.java

```diff
@@ -1,92 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.ignite.internal;
-
-import static com.tngtech.archunit.core.domain.JavaClass.Predicates.assignableTo;
-import static com.tngtech.archunit.lang.conditions.ArchPredicates.are;
-
-import com.tngtech.archunit.base.DescribedPredicate;
-import com.tngtech.archunit.core.domain.JavaClass;
-import com.tngtech.archunit.core.importer.ImportOption;
-import com.tngtech.archunit.junit.AnalyzeClasses;
-import com.tngtech.archunit.junit.ArchTest;
-import com.tngtech.archunit.lang.ArchCondition;
-import com.tngtech.archunit.lang.ArchRule;
-import com.tngtech.archunit.lang.ConditionEvents;
-import com.tngtech.archunit.lang.SimpleConditionEvent;
-import com.tngtech.archunit.lang.syntax.ArchRuleDefinition;
-import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
-import org.apache.ignite.configuration.ConfigurationModule;
-import org.apache.ignite.configuration.SuperRootChange;
-import org.apache.ignite.configuration.annotation.ConfigurationType;
-import org.apache.ignite.internal.configuration.ServiceLoaderModulesProvider;
-import org.apache.ignite.lang.LocationProvider.RootLocationProvider;
-
-/**
- * Tests that all local configuration modules do not override
- * {@link ConfigurationModule#patchConfigurationWithDynamicDefaults(SuperRootChange)}.
- */
-@AnalyzeClasses(
-        packages = "org.apache.ignite",
-        importOptions = ImportOption.DoNotIncludeTests.class,
-        locations = RootLocationProvider.class)
-public class ConfigurationModuleArchTest {
```

Comment:
Because now one local configuration module override patchConfigurationWithDynamicDefaults, so, the check is not actual anymore.

Reply:
Guys, I talked with @ibessonov about it, because the author is already left the project, I think. This test and appropriate javadocs were created just because of the fact, that at that time there were no cases for non-cluster-wide configurations. So, there are no any conceptual contradictions at ll.